test/cpp/end2end/nonblocking_test.cc

```diff
@@ -89,8 +89,9 @@ class NonblockingTest : public ::testing::Test {
     void* ignored_tag;
     bool ignored_ok;
     cq_->Shutdown();
-    while (LoopForTag(&ignored_tag, &ignored_ok))
+    while (LoopForTag(&ignored_tag, &ignored_ok)) {
       ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done

Reply:
Doesn't look like this was actually done.